### DISCUSSION TITLE
verdaccio  <=5.25.0   Depends on vulnerable versions of request

### DISCUSSION THREAD

**Discussion by CaneRandagio1983** [verdaccio/verdaccio#3918] [Q&A] [answered]
I am having the above error, how can I fix it?
```
andrea@Tehesyan:~/software/Evernote-for-Linux$ npm list verdaccio
andrea@ /home/andrea
└── verdaccio@5.25.0
```

**Comment by juanpicado** [comment, marked as answer]
~verdaccio 5 still uses `request` module, it has been removed for the alpha https://github.com/verdaccio/verdaccio/issues/1807~

~unfortunatelly you cannot fix anything here, just wait patiently next major is being released~

High probability is fixed on 5.x https://github.com/verdaccio/verdaccio/pull/3960

**Reply by juanpicado** [reply]
fyi https://github.com/verdaccio/verdaccio/releases/tag/v5.29.0
